grapher/controls/CollapsibleList.jsdom.test.tsx

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env yarn jest
+
+import * as React from "react"
+import { shallow } from "enzyme"
+import { CollapsibleList } from "./CollapsibleList"
+import { collapsibleListSampleItems } from "./CollapsibleList.sampleInput"
+
+// configure({ adapter: new Adapter() })
+describe(CollapsibleList, () => {
```

Comment:
A jsdom test is a great start! Maybe at somepoint we'll find a way to make these faster automagically